packages/ember-core/src/components/toaster.gts

```diff
@@ -0,0 +1,46 @@
+import { fn } from '@ember/helper';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import Alert from '../components/alert';
+import type ToastService from '../services/toast';
```

Comment:
My fault
```suggestion
import Alert from './alert.gts';
import type ToastService from '../services/toast.ts';
```

Reply:
You'll also need to resolve the import ordering manually, otherwise linting will fail